script.js

```diff
@@ -0,0 +1,139 @@
+let files;
+let currSection = "";
+const hostURL = 'https://api.jsonbin.io/v3/b/62fa3b83a1610e6386ff2396/';
+
+const loadContent = async () => {
+
+    let coursesSections = ["data-science", "drawing", "aws", "excel", "js", "python", "web-dev"];
+    
+    let response = await fetch(hostURL);
+    files = await response.json()
+
+    const container = document.querySelector('.courses-cards');
+    container.innerHTML = '';
+
+    loadCourses(coursesSections, container);
+} 
+
+function loadCourses(coursesSections, container){
+    container.innerHTML = '';
+
+    for(let file of coursesSections){
+        files['record'][file].courses.forEach(element => {
+            let inst = '';
+
+            element.instructors.forEach(instructor => {
+                inst += instructor.name + ', ';
+            });
+            
```

Comment:
remove blank lines between lines inside a function.
blank line should be between functions only
```suggestion
```